# HG changeset patch
# User Rik <rik@octave.org>
# Date 1424739749 28800
#      Mon Feb 23 17:02:29 2015 -0800
# Node ID 9b90c60033629439efc53cbb28f197191a589468
# Parent  9cf91ecdc6f73cdd74717fad813b1b54cfcb9cce
doc: Improve docstring for flipdim.

* flipdim.m: Improve docstring for flipdim.

diff --git a/scripts/general/flipdim.m b/scripts/general/flipdim.m
--- a/scripts/general/flipdim.m
+++ b/scripts/general/flipdim.m
@@ -18,18 +18,18 @@
 ## <http://www.gnu.org/licenses/>.
 
 ## -*- texinfo -*-
 ## @deftypefn  {Function File} {} flipdim (@var{x})
 ## @deftypefnx {Function File} {} flipdim (@var{x}, @var{dim})
 ## Flip array across specific dimension.
 ##
 ## This function is an alias for @code{flip} and exists for backwards
-## and @sc{matlab} compatibility.  See further details about its usage
-## on @code{flip} help text.
+## and @sc{matlab} compatibility.  See @code{flip} for complete usage
+## information.
 ##
 ## @seealso{flip, fliplr, flipud, rot90, rotdim}
 ## @end deftypefn
 
 ## Author: David Bateman, Jaroslav Hajek
 
 function y = flipdim (varargin)
   y = flip (varargin{:});
